.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  pull_request:
+    branches-ignore:
+      - main
+
+name: Validate
+
+jobs:
+  check:
+    name: Check Cherry Picks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check all commits exist in main
+        uses: cujomalainey/cherry-pick-check@v1
```

Comment:
Can you add a convenience link to https://github.com/cujomalainey/cherry-pick-check/blob/main/entrypoint.sh ?

Reply:
> Could you make https://github.com/cujomalainey/cherry-pick-check/blob/main/entrypoint.sh a bit more shellcheck-compliant?

Please criticize till your heart is content, this is the first serious shell script I have ever written

Reply:
I think the "native"/preferred language for Github Action is Typescript (superset of Javascript) if you prefer that.

One advantage of TypeScript is I won't be able to review it :-D

Reply:
@marc-hb if I write it typescript, neither of us will be able to read it